OcadoScraper.py

```diff
@@ -8,13 +9,24 @@
 class OcadoScraper:
     def __init__(self):
         """
-        Initialiser: to decide what functions are called here and what attributes and arguments are defined
-        """
+        Access Ocado front page using chromedriver
+        """
+        self.chrome_options = webdriver.ChromeOptions()
+        self.chrome_options.add_argument("--start-maximized")
+        self.driver = webdriver.Chrome(options=self.chrome_options)
+        self.driver.maximize_window()
+        self.driver.get('https://www.ocado.com/')
+        try:
```

Comment:
lines 19 - 22. Wondering if it would be better to have the try, except in the accept_cookies function? And then call ocado.accept_cookies() after initialising the object?

Reply:
That's what I did at first but then I thought it wouldn't be good to have a function which might not do anything. I am not sure what's best.